src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -62,8 +61,8 @@
 	public void propertiesCorrectlySet() {
 		new AstComparator(); // just to run the static code of the class
 
-		assertEquals(0.6, AbstractBottomUpMatcher.SIM_THRESHOLD, 1e-6);
-		assertEquals(1, AbstractSubtreeMatcher.MIN_HEIGHT);
+		// assertEquals(0.6, AbstractBottomUpMatcher.SIM_THRESHOLD, 1e-6);
+		// assertEquals(1, AbstractSubtreeMatcher.MIN_HEIGHT);
```

Comment:
Instead of doing so, it is better to use [`Ignore`](https://junit.org/junit4/javadoc/4.12/org/junit/Ignore.html). We will know in future if we need to revert this change.

Reply:
I will remove that commented code.